src/akushon/action/process/joint_process.cpp

```diff
@@ -50,6 +57,24 @@ void JointProcess::set_initial_position(float initial_position)
   this->initial_position = initial_position;
 }
 
+void JointProcess::interpolate_time(double time)
+{
+  bool time_is_reached = false;
+  double passed_time = time - initial_time;
+  double divider = passed_time / action_time;
+
+  time_is_reached |= (divider >= 1.0);
+
+  if (time_is_reached) {
```

Comment:
Unnecessary `time_is_reached` variable:
```suggestion
  double passed_time = time - initial_time;
  double divider = passed_time / action_time;

  if (divider >= 1.0) {
```